core/server/api_container/server/startosis_engine/startosis_runner.go

```diff
@@ -13,6 +15,8 @@ type StartosisRunner struct {
 	startosisValidator *StartosisValidator
 
 	startosisExecutor *StartosisExecutor
+
+	mutex *sync.Mutex
```

Comment:
we're adding this because

1. We don't support running two scripts against the same Enclave anyway
2. The Warning set can't be shared

Any other reasons? Can we add this as a comment?

Reply:
nope; those are the reasons - cool we add these.